ui/pages/confirmations/components/confirm/info/approve/edit-spending-cap-modal/__snapshots__/edit-spending-cap-modal.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<EditSpendingCapModal /> renders component 1`] = `<div />`;
```

Comment:
Looks like component is empty here

Reply:
There was an issue with rendering the portal. Mocking it fixed this.